jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/list/AbstractInstanceListViewTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.common.client.list;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.assertj.core.api.Assertions;
+import org.jbpm.workbench.common.model.GenericSummary;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.uberfire.ext.services.shared.preferences.GridGlobalPreferences;
+import org.uberfire.ext.widgets.table.client.ColumnMeta;
+
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+public abstract class AbstractInstanceListViewTest<T extends GenericSummary> extends AbstractMultiGridViewTest<T> {
+
+    @Test
+    public void addDomainSpecifColumnsTest() {
+        final ListTable<T> currentListGrid = spy(new ListTable<>(new GridGlobalPreferences()));
+        final Set<String> domainColumns = new HashSet<String>();
+        domainColumns.add("var1");
+        domainColumns.add("var2");
+        domainColumns.add("var3");
+        getView().addDomainSpecifColumns(currentListGrid,
+                                         domainColumns);
+
+        final ArgumentCaptor<List> argument = ArgumentCaptor.forClass(List.class);
+        verify(currentListGrid).addColumns(argument.capture());
+
+        final List<ColumnMeta> columns = argument.getValue();
+
+        Assertions.assertThat(columns.size()).isEqualTo(3);
```

Comment:
AssertJ tips:
1) I'd recommend using static import of `assertThat` to avoid having to write `Assertions.` prefix everywhere

2) `assertThat(columns.size()).isEqualTo(3)`  can be simplified to `assertThat(columns).hasSize(3);` which is more readable AND gives you better error message in case of assertion faiure)
This occurs a lot in this PR

3)
`assertThat(list.stream().map(.... getXYZ())).collect(toList())).containsAll(..)`
can be simplified to 
`assertThat(list).extracting("XYZ", Property.class).containsAll(...)`

See this example

```java
        assertThat(Arrays.asList(new User("John"), new User("Jane")))
                .extracting("name", String.class)
                .containsExactly("John", "Jane");
  

    static class User {

        private String name;

        public User(String name) {
            this.name = name;
        }

        public String getName() {
            return name;
        }
 ```
This also occurs many times in this PR.

Reply:
ok, Thank you !